cmd/ftl/cmd_box_run.go

```diff
@@ -2,13 +2,14 @@ package main
 
 import (
 	"context"
-	"database/sql"
 	"fmt"
 	"net/url"
 	"time"
 
+	"github.com/XSAM/otelsql"
 	_ "github.com/jackc/pgx/v5/stdlib" // pgx driver
 	"github.com/jpillora/backoff"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
```

Comment:
We have a test  in internal/observability/client_test.go to ensure the semconv version is consistent that you'll need to update to mention this file, _or_ alternatively change the wording to be more generalised.